src/treeView.ts

```diff
@@ -45,7 +45,7 @@ function makeModule(moduleName: string, elementId?: string): LayerItem {
 
 function makeDependency(dependencyName: string, elementId?: string): LayerItem {
 	const layerItem = new LayerItem(dependencyName, 'furyDependency', elementId);
-	layerItem.iconPath = makeIconPath('link-external');
+	layerItem.iconPath = makeIconPath('link');
```

Comment:
We also need to put `link.svg` icon in [icons](https://github.com/propensive/fury-vscode/tree/master/media/icons) directory. For now, this directory contains only icons that are currently used.

Reply:
Ah, I see. I'll do that.